src/adt.c

```diff
@@ -320,7 +320,7 @@ int adt_get_reg(const void *adt, int *path, const char *prop, int idx, u64 *padd
         u32 pa_cells = 2;
         ADT_GETPROP(adt, parent, "#address-cells", &pa_cells);
 
-        dprintf(" translate range to address-cells=%d size-cells=%d\n", pa_cells, ps_cells);
+        dprintf(" translate range to address-cells=%d\n", pa_cells);
```

Comment:
all the changes in `src/` seems unrelated and included by accident

Reply:
Fixed.